ERCS/erc-TBA.md

```diff
@@ -0,0 +1,128 @@
+---
+title: Soulbound Degradable Governance
+description:
+  A governance standard for DAOs where voting power is non-transferable and decays over time without
+  active contributions.
+author: Guilherme Neves (@0xneves) Rafael Castaneda (@castacrypto)
+discussions-to: https://ethereum-magicians.org/t/soulbounded-degradable-governance-sdg-a-new-approach-to-dao-power-structures/21326
```

Comment:
```suggestion
discussions-to: https://ethereum-magicians.org/t/erc-7787-soulbound-degradable-governance/21326
```
I updated the Eth Magicians title to add in the ERC number and match the title of the ERC